src/components/file-selector/file-selector.stories.ts

```diff
@@ -175,6 +194,42 @@ export const DefaultOnlyPDF: StoryObj = {
   args: { ...defaultArgs, accept: '.pdf' },
 };
 
+export const DefaultMultiSizeS: StoryObj = {
```

Comment:
Could we maybe remove some of the new stories and still have every change covered by Chromatic?

Reply:
Maybe I could use the `excludeStories` param in the Meta object, as in the button stories? or do you have other ideas?

Reply:
No, then it's ok if they are public. but looking at the code, `size` influences the buttons, so if we use the example where everything is visible, would that not be enough?

Reply:
I added a function to display the file list in chromatic (it needs to be tested, i pushed right now) and I removed the stories with the error, since it's not influenced by the size change